iolog.c

```diff
@@ -485,16 +492,48 @@ static bool read_iolog2(struct thread_data *td, FILE *f)
 	return true;
 }
 
+#ifndef S_ISSOCK
+#define S_ISSOCK(x) 0
+#endif
+
+static bool is_socket(const char* path) {
```

Comment:
Fio style is to place the * next the variable, and the opening brace on the next line (except for if). This should be:

```static bool is_socket(const char *path)
{
[...]
```

and ditto for other locations.